File: web-shop/src/main/webapp/js/service/seckill_orderService.js
//服务层
app.service('seckill_orderService',function($http){

	//查询实体
	this.findOne=function(id){
		return $http.get('../seckill_order/findOne.do?id='+id);
	}
	//取消订单
	this.dele=function(id){
		return $http.get('../seckill_order/delete.do?id='+id);
	}
    //确认已发货
    this.updateStat=function(ids){
        return $http.get('../seckill_order/updateStat.do?ids='+ids);
    }
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../seckill_order/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: interface/src/main/java/cn/itcast/core/service/seckill_orderService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.entity.PageResult;
import cn.itcast.core.pojo.seckill.SeckillOrder;
import org.springframework.stereotype.Repository;

@Repository
public interface Seckill_orderService {


    //分页加条件查询
    PageResult search(Integer page, Integer rows, SeckillOrder seckillOrder);

    //点修改回显数据
    SeckillOrder findOne(Long id);

    //修改
    void update(SeckillOrder seckillOrder);

    //取消订单
    void delete(Long id);

    //批量修改状态
    void updateStat(Long[] ids);

}
